cdap-spark-core-base/src/main/java/io/cdap/cdap/app/runtime/spark/SparkProgramRuntimeProvider.java

```diff
@@ -93,12 +95,21 @@ public ClassLoader createProgramClassLoader(CConfiguration cConf, ProgramType ty
       cConf.getBoolean(SparkRuntimeUtils.SPARK_STREAMING_CHECKPOINT_REWRITE_ENABLED);
     boolean rewriteYarnClient = cConf.getBoolean(Constants.AppFabric.SPARK_YARN_CLIENT_REWRITE);
 
+    ClassLoader sparkClassLoader = null;
```

Comment:
Call this `sparkRunnerClassLoader` to be consistent with another method.

Reply:
Fixed